lib/iris/experimental/ugrid/mesh.py

```diff
@@ -560,10 +562,10 @@ def has_lazy_indices(self):
         """
         return super()._has_lazy_values()
 
-    def lazy_src_lengths(self):
+    def lazy_location_lengths(self):
         """
         Return a lazy array representing the lengths of each
-        :attr:`src_location` in the :attr:`src_dim` of the connectivity's
+        :attr:`location_element` in the :attr:`location_axis` of the connectivity's
```

Comment:
As above, replace with 'location element', also consider replacing "lengths of each" with "lengths associated with each".

Reply:
b01341dce90fb5f1e2c7d33aa5d44152af2924a7